overrides/browsers/chrome-override.json

```diff
@@ -1,5 +1,60 @@
 {
     "features": {
+        "apiManipulation": {
+            "state": "enabled",
```

Comment:
Should we do this for Edge too?

Reply:
Yeah possibly. I think we're going to have to move this to Monday as shipping this on Friday isn't super wise.

Reply:
Yeah good call.